contracts/src/Teleporter/upgrades/TeleporterUpgradeable.sol

```diff
@@ -217,6 +221,33 @@ abstract contract TeleporterUpgradeable is Context, ITeleporterReceiver, Reentra
      */
     function _checkTeleporterUpgradeAccess() internal virtual;
 
+    /**
+     * @dev Sends a cross chain message using the TeleporterMessenger contract.
+     * @return `messageID` The unique identifier for the Teleporter message.
+     * @return `adjustedFeeAmount` The adjusted fee amount after transferring the fee token.
```

Comment:
Is there any need to return the `adjustedFeeAmount` here? It seems a little odd that the interface doesn't match `sendCrossChainMessage`.

Reply:
hmm I originally just noticed in our example apps we like to emit in an event the `adjustedFeeAmount`, but that makes sense, I'll just return `sendCrossChainMessage` call